packages/mask/src/plugins/Predict/Azuro/api/index.ts

```diff
@@ -0,0 +1,71 @@
+import { calculateActualOdds } from '@azuro-protocol/sdk'
+import type { ChainId } from '@masknet/web3-shared-evm'
+import { configureAzuroSDK } from '../helpers/configureAzuroSDK'
+import type { GamesRaw, OddsByConditions, RawEvents } from '../types.js'
+
+export async function fetchOddsByConditions(conditionIds: number[]): Promise<OddsByConditions> {
+    const oddsData = await fetch('https://api.bookmaker.xyz/odds', {
```

Comment:
Declare the `host_permissions` with this API host to force users to click a approve button before revealing the plugin UI. The browser will not restrict us from accessing resources from this host in the long-term future.

Here is am example: packages/mask/src/plugins/Gitcoin/base.ts